lib/web3/formatters.js

```diff
@@ -159,6 +159,9 @@ var inputPostFormatter = function(post) {
     post.priority = utils.fromDecimal(post.priority);
 
     // fallback
+    if (post.topics == undefined) {
+        post.topics = []
+    }
     if (!utils.isArray(post.topics)) {
```

Comment:
this is the contents of post.topics: `[ undefined ]` if I don't specify any in my code.